git_basics/js_basics/task4.js

```diff
@@ -0,0 +1,4 @@
+let age = 10;
+
+const adult = age > 18;
```

Comment:
name for boolean variables should always have a question form  (isMarried, hasChildren etc)